package.json

```diff
@@ -18,15 +18,21 @@
     "e2e:cloud": "E2E_RUN_IN_CLOUD=true npm run e2e",
     "e2e:tunnel": "E2E_BASE_URL=http://localhost:3000 E2E_USE_TUNNEL=true npm run e2e:cloud",
     "e2e:local": "E2E_BASE_URL=http://localhost:3000 npm run e2e",
+    "jest": "./node_modules/jest/bin/jest.js",
+    "jest:update": "./node_modules/jest/bin/jest.js -u",
     "setup": "npm run apidocs:setup && npm run apidocs:build && npm run synapse:setup && gem install bundler && bundle install && npm install",
     "server": "node ./server/index.js",
     "server:dev": "NODE_ENV=development npm run server",
     "server:prod": "NODE_ENV=production npm run server",
     "start": "npm run server:prod",
     "synapse:setup": "git submodule init && git submodule update && cd synapse && npm ci",
-    "test:server": "cross-env NODE_ENV=production ./node_modules/.bin/mocha ./server/index.spec.js --timeout=10000"
+    "test:server": "cross-env NODE_ENV=production ./node_modules/.bin/mocha ./server/index.spec.js --timeout=10000",
+    "vue:build": "cross-env NODE_ENV=production webpack --progress --hide-modules",
+    "vue:dev": "cross-env NODE_ENV=development webpack",
+    "vue:watch": "cross-env NODE_ENV=development webpack --watch"
```

Comment:
Not related to your change at all, but while looking at this it got me thinking. I find these hard to remember. What do you think about updating these to follow a format like`server:node:dev` and `server:jekyll:public`, or even simpler: `node:dev`, `jekyll:public`. I'd be happy to make the change in another ticket if you're alright with that.

Reply:
I like that idea, but since this PR already has quite a lot in it, would prefer to handle in a separate ticket